docs/edgekv_cli.md

```diff
@@ -392,6 +393,25 @@ Usage:
 | - | - | - |
 | tokenName | required | token name |
 
+### List Groups in Namespace
+
+List the groups for a given namespace in an Akamai environment.
+
+Usage:
+`akamai edgekv list  groups <environment> <namespace>`
+
+Example:
+`akamai edgekv list  groups  production default `
+
+| Option | Description |
+| - | - |
+| -h, --help  | Display information on how to use this EdgeKV command. |
+
+| Argument | Existence | Description |
+| - | - | - |
+| environment | required | The Akamai environment from which to retrieve a list of groups, either “staging” or “production”. |
+| namespace | required | Namespace identifier |
+
```

Comment:
Sarah also suggested we add " This operation returns up to 10,000 groups. If the namespace contains more than 10,000 groups, the operation returns a random set of 10,000 groups. ".
Please follow up with her as to where exactly this should be added (e.g. in the description or help or elsewhere)

Reply:
Thank you medhak . I will follow with sarah . this was pull request I have created to see my local repo rebase issues.I will create new one with your updated text suggestions and sarah's.